planckton/sim.py

```diff
@@ -139,7 +139,7 @@ def __init__(
     def run(self):
         """Run the simulation."""
         hoomd_args = f"--single-mpi --mode={self.mode}"
-        sim = hoomd.context.initialize(hoomd_args)
+        sim = hoomd.Simulation(device=hoomd_args)
```

Comment:
In hoomd3, the device object is its own class, so we might want to do something like:

device = hoomd.device.auto_select()
sim = hoomd.Simulation(device=device)